blog/2024-07-31-open-planning-r24.12.mdx

```diff
@@ -0,0 +1,122 @@
+
+:::info
+The detailed agenda will follow as soon as possible
+:::
+
+### Refinement Day 1 - Release 24.12
+
+This event is organised and moderated by the Catena-X Association. Based on a predefined agenda (depending on the features on the board), individual dependencies are discussed and
+also documented directly on the feature. If the refinement of the features has already been carried out 'in the refinement phase' and the Definition of Entry (DoE) has already reached,
+this meeting can also be used as synchronisation.
+
+- *Monday, 1. July 2024 from 09:05 to 12:00 (CEST)*
+- [open meeting](https://eclipse-tractusx.github.io/community/open-meetings/#Refinement%20Day%201%20-%20Release%2024.12)
+
+:::info
+The detailed agenda will follow as soon as possible
+:::
+
+### Draft Feature Freeze - Release 24.12
+
+This event is organised and moderated by the Catena-X Association. Until this date, feature proposal can be submitted to the planning board (in GitHub). The current status (as an
+interim status in the refinement phase) of the previous feature proposals will also be discussed at this meeting.
+
+- *Friday, 12. July 2024 from 09:05 to 12:00 (CEST)*
+- [open meeting](https://eclipse-tractusx.github.io/community/open-meetings/#Draft%20Feature%20Freeze%20-%20Release%2024.12)
+
+:::info
+The detailed agenda will follow as soon as possible
+:::
+
+### Refinement Day 2 - Release 24.12
+
+This event is organised and moderated by the Catena-X Association. Based on a predefined agenda (depending on the features on the board), individual dependencies are discussed and
+also documented directly on the feature. If the refinement of the features has already been carried out 'in the refinement phase' and the Definition of Entry (DoE) has already reached,
+this meeting can also be used as synchronisation.
+
+- *Monday, 22. July 2024 from 09:05 to 12:00 (CEST)*
+- [open meeting](https://eclipse-tractusx.github.io/community/open-meetings/#Refinement%20Day%202%20-%20Release%2024.12)
+
+:::info
+The detailed agenda will follow as soon as possible
+:::
+
+### Open Planning Days - Release 24.12
+
+Open Planning, hosted by the Catena-X association. The goal for this meeting is planning of the intended work content for the next release - which contains also preparation of future
+release content, refactoring, general architectural work, tool & process improvements and feature planning for the next Tractus-X release.
+
+- *Wednesday, 31. Jul 2024 and Thursday, 01. Aug 2024 from 08:05 to 12:00 (CEST)*
+- [open meeting](https://eclipse-tractusx.github.io/community/open-meetings/#Open%20Planning%20Days%20-%20Release%2024.12)
+
+:::info
+The detailed agenda will follow as soon as possible
+:::
+
+## Staying Informed
+
+To stay updated on any changes or additional information, please subscribe to our mailing list. A great way to keep informed is through the provided [Tractus-X Development Mailing List RSS Feed](https://www.eclipse.org/lists/tractusx-dev/maillist.rss).
```

Comment:
"A great way to STAY informed"?

Reply:
I am not sure if you meant the header … I changed it because I like your header more than mine.

Reply:
Oh good point too, but I actually meant the second sentence, where it says "keep informed" and I think "stay informed" is better 😁

Reply:
adapted, thx :)